### PR TITLE
Added platinum bluetooth elements post

### PR DESCRIPTION
**Pull request by beaufortfrancois**
As suggested by @tjsavage, here's a draft announcing the platinum bluetooth elements.

### CODE REVIEW

File: _posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+<a href="https://elements.polymer-project.org/elements/platinum-bluetooth?view=demo:demo/">
+  <img src="/images/releases/platinum-bluetooth-demo.png" style="width: auto; margin-left: auto; margin-right: auto; height: 450px" alt="platinum bluetooth demo screenshot">
+</a>
+
+## Scan for Bluetooth Devices
+
+It is worth mentioning again there that due to [security
+requirements](https://developers.google.com/web/updates/2015/07/interact-with-ble-devices-on-the-web#https-only),
+Web Bluetooth will only work for HTTPS websites.
```
Comment (jeffposnick) [inline _posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown]: You can plug https://github.com/PolymerElements/platinum-https-redirect here.

Comment (addyosmani) [inline _posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown]: :+1:

Comment (beaufortfrancois) [inline _posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown]: Fixed! Thank you @jeffposnick

File: _posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+author: fbeaufort
+categories: announcements
+excerpt: A new set of Polymer elements that simplify discovering and communicating with nearby Bluetooth devices, powered behind the scenes by the experimental Web Bluetooth API is now available.
+
+---
+
+I'm particularly glad to share with you today the [platinum bluetooth elements](https://elements.polymer-project.org/elements/platinum-bluetooth?active=platinum-bluetooth-device), a
+brand new set of [Polymer platinum
+elements](https://elements.polymer-project.org/browse?package=platinum-elements)
+to discover and communicate with nearby Bluetooth devices powered behind the
```
Comment (tjsavage) [inline _posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown]: Might make this two sentences - e.g. "...to dicsover and communicate with nearby Bluetooth devices. The new components are powered behind the scenes by the experimental..."

Comment (beaufortfrancois) [inline _posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown]: Done.

**Comment by beaufortfrancois** [comment]
@tjsavage Comment has been addressed. 
Feel free to merge at your convenience.

**Pull request merged**